src/utils.js

```diff
@@ -132,8 +132,19 @@ export default {
     return json;
   },
 
-  async getImageSize(url: string): Promise<PageDimensions> {
-    const { width, height } = await probe(url);
+  async getImageSize(
+    url: string,
+    options: ?Object,
+  ): Promise<{ width: ?number, height: ?number }> {
+    let width;
+    let height;
+
+    try {
```

Comment:
Wrapping this in a `try` block since timeouts throw a `ETIMEDOUT` error which we don't want to pass to clients. We just want to give up on getting image sizes.